code/game/machinery/hologram.dm

```diff
@@ -50,6 +50,11 @@ Possible to do for anyone motivated enough:
 	var/static/force_answer_call = FALSE	//Calls will be automatically answered after a couple rings, here for debugging
 	var/static/list/holopads = list()
 
+/obj/machinery/holopad/examine(mob/user)
+	..()
+	if(isAI(user))
+		to_chat(user, "<span class='notice'>Alt-click to hang up all calls.</span>")

```

Comment:
I couldn't figure out if the AI can examine things. Can it? If not, how can it figure out how to use alt-click with the holopad?

Reply:
it can't examine

Reply:
It should be able to for nearby objects [from core], although this will be largely useless.

Reply:
yeah ai's cant examine due to being a shitty camera

Reply:
Ok thanks, I'll remove this one.